src/server/openapi.yml

```diff
@@ -0,0 +1,446 @@
+        FOCUS_SPECIFICATIES:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        FOCUS_TOZO:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        FOCUS_TONK:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        FOCUS_BBZ:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        FOCUS_STADSPAS:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        WMO:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        VERGUNNINGEN:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        MY_LOCATION:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        AFVAL:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        AFVALPUNTEN:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        BELASTINGEN:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        MILIEUZONE:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        TOERISTISCHE_VERHUUR:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        ERFPACHT:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        SUBSIDIE:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        NOTIFICATIONS:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+        CASES:
+          oneOf: [$ref: '#/components/schemas/ApiSearchConfigResult']
+    ApiSearchConfigResult:
+      type: object
+      properties:
+        keywords:
+          type: array
+          items: {}
+        keywordsGeneratedFromProps:
+          type: array
+          items: {}
+        description:
+          type: string
+    ServicesAllResponse:
+      type: object
+      properties:
+        AFVAL:
+          type: object
+        AFVALPUNTEN:
+          type: object
+        BRP:
+          type: object
+        BELASTINGEN:
+          type: object
+        CMS_CONTENT:
+          type: object
+        CMS_MAINTENANCE_NOTIFICATIONS:
+          type: object
+        ERFPACHT:
+          type: object
+        KREFIA:
+          type: object
+        FOCUS_AANVRAGEN:
+          type: object
+        FOCUS_SPECIFICATIES:
+          type: object
+        FOCUS_TOZO:
+          type: object
+        FOCUS_TONK:
+          type: object
+        FOCUS_BBZ:
+          type: object
+        FOCUS_STADSPAS:
+          type: object
+        NOTIFICATIONS:
+          type: object
+        CASES:
+          type: object
+        MY_LOCATION:
+          type: object
+        KVK:
+          type: object
+        MILIEUZONE:
+          type: object
+        TOERISTISCHE_VERHUUR:
+          type: object
+        SUBSIDIE:
+          type: object
+        VERGUNNINGEN:
+          type: object
+        WMO:
+          type: object
+    StreamResponse:
+      type: string

```

Comment:
Kun je hier iets maken als: https://github.com/OAI/OpenAPI-Specification/issues/396#issuecomment-894718960

Reply:
Heb het aangepast, een vervolgstap zou nog kunnen zijn de specifieke service berichten uit te werken en op te nemen in de config.